portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/AdvancedObjectAPI/AdvancedObjectAPI.testcase

```diff
@@ -43,6 +44,42 @@ definition {
 		}
 	}
 
+	@description = "Aggregation Terms in oneToMany relationship"
+	@priority = "5"
+	test CanReturnAggregationTermsDetailsInOneToManyRelationship {
+		property portal.acceptance = "true";
+
+		task ("Given two manager accounts created") {
+			var managerId1 = ObjectDefinitionAPI.createManager(managerFirstname = "Courtney");
+			var managerId2 = ObjectDefinitionAPI.createManager(managerFirstname = "Aaron");
+		}
+
+		task ("Given three employee accounts created") {
+			var employeeId1 = ObjectDefinitionAPI.createEmployee(
+				employeeFirstname = "Bruce",
+				managerId = "${managerId1}");
+			var employeeId2 = ObjectDefinitionAPI.createEmployee(
+				employeeFirstname = "Riley",
+				managerId = "${managerId2}");
+			var employeeId3 = ObjectDefinitionAPI.createEmployee(
+				employeeFirstname = "Justin",
+				managerId = "${managerId2}");
+		}
+
+		task ("When calling for 'employees' with a 'managerId' as aggregationTerms parameter") {
+			var getFacetsWithManagerId = ObjectDefinitionAPI.getObjectsWithAggregationTerms(
+				facetCriteria = "managerId",
+				objects = "employees");
+		}
+
+		task ("Then I receive the managerId in facets with a correct number of ocurrences") {
+			ObjectDefinitionAPI.assertInFacetsWithCorrectValue(
+				expectedNumberList = "2,1",
+				expectedTermList = "${managerId2},${managerId1}",
```

Comment:
how can we make sure that first number is for first managerID and the second number for the other one? 
Maybe in `macro assertInFacetsWithCorrectValue` we could check both at once?
For example with a pattern similar to this one `JSONUtil.getWithJSONPath("${responseToParse}", "$..items[?(@.id==${employeeId2} && @.r_supervisor_c_managerId==${managerId2})].r_supervisor_c_managerId");`